packages/cozy-sharing/src/components/ShareModal.jsx

```diff
@@ -32,12 +32,11 @@ export const ShareModal = ({
 }) => {
   const showShareCozyToCozyForNotes =
     flag('notes.sharing-cozy-to-cozy') !== null &&
-    flag('notes.sharing-cozy-to-cozy') &&
-    documentType === 'Notes'
+    flag('notes.sharing-cozy-to-cozy')
 
   const showShareByEmail =
-    showShareCozyToCozyForNotes &&
-    documentType !== 'Albums' &&
+    ((documentType === 'Notes' && showShareCozyToCozyForNotes) ||
```

Comment:
I think we should definitely make the conditional more readable from a business point of view

Reply:
@acezard we wrote a new version with @Ldoppea and @cballevre